bigtable/bigtable-proxy/src/main/java/com/google/cloud/bigtable/examples/proxy/commands/Verify.java

```diff
@@ -222,6 +218,7 @@ void checkMetrics(Credentials creds) throws IOException {
     CompletableResultCode result = exporter.export(metricData);
     result.join(1, TimeUnit.MINUTES);
 
+    System.out.println("Metrics resource: " + resource);
```

Comment:
should we remove this line?

Reply:
No, I want it to print the detect monitored resource